### PR TITLE
Diagram database

### PR DESCRIPTION
**Pull request by J-C-S-V**
Hello dear reviewer 👋, I hope you're having a nice day today 😊

This is what we did on this branch:

- Create a file named schema_based_on_diagram.sql.
- Join tables from many-to-many relationships.
- Add the FK indexes.

Regards! ✨

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+/* Join table for medical_histories(id) & treatments(id) */
+CREATE TABLE medical_history_treatments (
+    medical_history_id INT,
+    treatment_id INT,
+    PRIMARY KEY (medical_history_id, treatment_id),
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
```
Comment (omar-muhamad) [inline schema_based_on_diagram.sql:55]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=still%20need%20them.-,Remember%20to%20add%20the%20FK%20indexes,-.), Please add indexes for all foreign keys you have at the end of the schema file.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @J-C-S-V and @VelzckC0D3,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (VelzckC0D3) [reply schema_based_on_diagram.sql:55]: @Omar-Muhamad Done! thank you for noticing 🙌

**Review by Olamarx — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Added Foreign key indexes ✔️
- Create join table ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**